Makefile

```diff
@@ -13,3 +13,7 @@ gen-doc:
 
 gen-docs:
 	docker run -v $(shell pwd):/go/src/app sigdocs
+
+test-unit:
```

Comment:
It's good to have a dockerized path, but a regular `go test ./generator/...` should work too. it currently leaves a dirty path.